great_expectations/util.py

```diff
@@ -1489,3 +1490,28 @@ def pandas_series_between_inclusive(
         metric_series = series.between(min_value, max_value, inclusive=True)
 
     return metric_series
+
+
+def numpy_quantile(
+    a: np.ndarray, q: float, method: str, axis: Optional[int] = None
+) -> np.ndarray:
+    """
+    As of NumPy 1.21.0, the 'interpolation' arg in quantile() has been renamed to `mode`.
```

Comment:
@cdkini Not `mode`, but `method`, correct?  Thanks.

Reply:
you are right!